descriptors/ModuleDescriptor-template.json

```diff
@@ -69,7 +69,7 @@
     },
     {
       "id": "loan-policy-storage",
-      "version": "2.0",
+      "version": "3.0",
```

Comment:
@marcjohnson-kint. not sure if this a breaking change, yep I only added the validation rules. However,  in case a consumer will specify `alternateRenewalLoanPeriod` or `RenewalsPolicy/period` for fixed policies that will not work longer.

Reply:
As I understand it, from an interface definition perspective, it is legitimate to define a policy with this policy, in the sense that nothing within the interface stops this from happening.

Given that it is not possible to express this kind of conditional validation within the interface definition (at the moment), I think this is an implementation breaking change, rather than an interface compatibility breaking change.

That means that this change should change the implementation version (in the pom.xml, from 9.x to 10.0.0 I believe) in this pull request (and in the JIRA issue).

Does that make sense?

@skoczko do you agree with this evaluation?

Reply:
done